packages/sol6/src/yang/etsi-nfv-vnf.yang

```diff
@@ -1374,10 +1374,7 @@ submodule etsi-nfv-vnf {
                  information element.";
             }
             leaf type {
-              type enumeration {
-                enum affinity;
-                enum anti-affinity;
-              }
+              type affinity-type;
```

Comment:
Call it 'type' and have it use the typedef for 'affinity-type'.

Reply:
Isn't that whai I am doing here? the leaf is called type, and i use the type 'affinity-type' which is a typedef?